src/herder/HerderImpl.cpp

```diff
@@ -949,8 +949,16 @@ HerderImpl::triggerNextLedger(optional<uint32_t> ledgerSeqToTrigger,
     // We pick as next close time the current time unless it's before the last
     // close time. We don't know how much time it will take to reach consensus
     // so this is the most appropriate value to use as closeTime.
+    auto const usingManualCloseWithoutExplicitCloseTime =
+        mApp.getConfig().MANUAL_CLOSE && !minCloseTime;
+    auto static const manualCloseDefaultPerLedgerTimeTIncrement =
+        VirtualClock::to_time_t(
+            std::chrono::system_clock::time_point(std::chrono::seconds{5}));
     uint64_t nextCloseTime =
-        VirtualClock::to_time_t(mApp.getClock().system_now());
+        usingManualCloseWithoutExplicitCloseTime
```

Comment:
More generally, does this code even need to be in `triggerNextLedger` or could it be in `manualClose` (transmitted via the `minTime` parameter)?

Reply:
It has, indeed, since been moved out of `triggerNextLedger()` and into `manualClose()`.